src/Polly.RateLimiting/RateLimiterRejectedException.cs

```diff
@@ -38,13 +38,33 @@ public RateLimiterRejectedException(string message)
     {
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="RateLimiterRejectedException"/> class.
+    /// </summary>
+    /// <param name="message">The message that describes the error.</param>
+    /// <param name="telemetrySource">The source pipeline and strategy names.</param>
```

Comment:
Let's just make this something like "A string representing the source of the telemetry". The format is opaque to the exception itself.